src/Salt/UserBundle/Controller/SignupController.php

```diff
@@ -92,6 +93,18 @@ public function signupAction(Request $request)
                     // Do not throw an error to the client if the email could not be sent
                 }
 
+                // send email to admin about this user creation
+                // get public users username and org
+                try {
+                    $from_email = $this->getParameter('mail_feature_from_email');
+                    $command = new SendAdminNotificationEmailCommand($from_email, $targetUser->getUsername(), $targetUser->getOrg()->getName());
```

Comment:
Can we assume that the `mail_feature_from_email` is the same email address that should receive notifications about new users?

Can be left for later, but it should probably be documented in an issue so that it is not forgotten about.

Reply:
Added an issue https://github.com/opensalt/opensalt/issues/406